docs/src/utils/Slider.js

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+import * as React from "react";
+import styled from "styled-components";
+
+const SliderInput = styled.input.attrs({ type: "range" })`
+  width: 120px;
+  margin-right: 20px;
+  appearance: none;
+  background: transparent;
+  &:focus {
+    outline: 0;
+  }
+  &::-webkit-slider-thumb {
+    appearance: none;
+    height: 12px;
+    width: 28px;
+    border-radius: 6px;
+    background: #edcc28;
+    cursor: pointer;
+    &:hover {
+      opacity: 0.5;
+    }
+  }
+  &::-webkit-slider-runnable-track {
+    width: 100%;
+    height: 16px;
+    cursor: pointer;
+    background: transparent;
+    border: 1px solid #edcc28;
+    border-radius: 8px;
+    padding: 1px;
+  }
+`;
+
+const SWrapper = styled.div`
+  display: flex;
+  align-items: center;
+`;
+
+const SEndpoint = styled.span`
+  font-size: 14px;
+  margin-left: 8px;
+  margin-right: 8px;
+`;
+
+type Props = {
+  onChange: (number) => void,
+  min?: number,
+  max?: number,
+  minLabel: boolean | string,
+  maxLabel: boolean | string,
+};
+
+export default function Slider({ onChange, minLabel, maxLabel, ...props }: Props) {
```

Comment:
Will be nice to show the slider value on hover.

Reply:
+1, I agree but I want to let @edenh figure out how it should look :)  (Another option is just to use `Scrubber` everywhere instead?)

Reply:
I like that the slider shows min and max. If you add a value indicator at the bottom of the scrubber (similar to progress bar) we could totally use it everywhere.